.vsts-ci.yml

```diff
@@ -126,18 +126,18 @@ stages:
 
     - template: /eng/build.yml
       parameters:
-        agentOs: Ubuntu_18_04
+        agentOs: Ubuntu_20_04
         pool:
           ${{ if eq(variables['System.TeamProject'], 'public') }}:
             name: NetCore1ESPool-Svc-Public
-            demands: ImageOverride -equals Build.Ubuntu.1804.Amd64.Open
+            demands: ImageOverride -equals 1es-ubuntu-2004-open
           ${{ if ne(variables['System.TeamProject'], 'public') }}:
             name: NetCore1ESPool-Svc-Internal
-            demands: ImageOverride -equals Build.Ubuntu.1804.Amd64
+            demands: ImageOverride -equals 1es-ubuntu-2004
         ${{ if eq(variables['System.TeamProject'], 'public') }}:
-          helixTargetQueue: Ubuntu.1804.Amd64.Open
+          helixTargetQueue: 'ubuntu.2004.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-20.04-helix-amd64-20220502145738-4b2e4c2'
         ${{ if ne(variables['System.TeamProject'], 'public') }}:
-          helixTargetQueue: Ubuntu.1804.Amd64
+          helixTargetQueue: Ubuntu.2004.Amd64
```

Comment:
You're also not using the docker tag here; whichever you decide you need should probably be in both places